normandy/base/middleware.py

```diff
@@ -17,6 +19,7 @@
 
 
 logger = logging.getLogger(__name__)
+metrics = markus.get_metrics("normandy")
```

Comment:
Perhaps this should be the value of `statsd_namespace`, and it should default to "normandy"? I think I'd be surprised if i changed METRICS_STATSD_NAMESPACE and that value was prepended to "normandy" instead of replacing it.

I'm not sure we'd ever need to change the namespace, so you could just drop that option too if you like.

Reply:
I wasn't actually sure what `statsd_namespace` did. Is it just a prefix to the metric name? I didn't see any info about that in the Markus docs. If so I think making it default to `normandy` would work well.